opencraft/settings.py

```diff
@@ -295,6 +295,10 @@
 # for provisioning Open edX app servers.
 DEFAULT_CONFIGURATION_VERSION = env('DEFAULT_CONFIGURATION_VERSION', default=DEFAULT_OPENEDX_RELEASE)
 
+# Git tag for latest Open edX release. Used as a default refspec for
+# configuration, edx-platform, forum, notifier, xqueue, and certs when creating production instances.
+LATEST_OPENEDX_RELEASE = env('LATEST_OPENEDX_RELEASE', default='named-release/dogwood.rc')
```

Comment:
@antoviaque I introduced this setting as part of https://github.com/open-craft/opencraft/pull/71. Changing the name to `OPENEDX_RELEASE_STABLE_REF` is a good idea - done in https://github.com/open-craft/opencraft/pull/71/commits/e200b556707ad8e31608b510552aa41dbb2a307a.